Companion patch to xtradeb-rust-zlib-unstable.patch .

Warning: quilt(1) will not refresh this patch correctly.

diff --git a/third_party/rust/zlib-rs/.cargo-checksum.json b/third_party/rust/zlib-rs/.cargo-checksum.json.old
rename from third_party/rust/zlib-rs/.cargo-checksum.json
rename to third_party/rust/zlib-rs/.cargo-checksum.json.old

diff --git a/third_party/rust/zlib-rs/.cargo-checksum.json b/third_party/rust/zlib-rs/.cargo-checksum.json
new file mode 100644
--- /dev/null
+++ b/third_party/rust/zlib-rs/.cargo-checksum.json
@@ -0,0 +1 @@
+{"files":{},"package": null}
